custom/rx_addendum.txt

```diff
@@ -1,3 +0,0 @@
-Medical Director of XXXX Clinics
```

Comment:
If get no direction on this, would bump this to another PR request(then will spend some more time on analyzing the code where these globals are used to see if will be useful to have as global settings).